osf/models/preprint_provider.py

```diff
@@ -15,7 +15,7 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     name = models.CharField(null=False, max_length=128)  # max length on prod: 22
     logo_name = models.CharField(null=True, blank=True, max_length=128)  # max length on prod: 17
     header_text = models.TextField(default='', blank=True)
-    description = models.CharField(null=True, blank=True, max_length=256)  # max length on prod: 56
+    description = models.TextField(null=True, blank=True)
```

Comment:
Removed. Forgot that you pointed that out last time since nearly everything in this model is still nullable.